patches/api/0449-Add-required-plugins-config-option.patch

```diff
@@ -0,0 +1,44 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Tamion <70228790+notTamion@users.noreply.github.com>
+Date: Sun, 26 Nov 2023 09:02:09 +0100
+Subject: [PATCH] Add required-plugins config option
+
+
+diff --git a/src/main/java/org/bukkit/Bukkit.java b/src/main/java/org/bukkit/Bukkit.java
+index 4863d9f21f0a0f11974be85360edc587ffd7eab3..06652c0add4ff5d2d2a0467ae5c95d1511203610 100644
+--- a/src/main/java/org/bukkit/Bukkit.java
++++ b/src/main/java/org/bukkit/Bukkit.java
+@@ -985,6 +985,15 @@ public final class Bukkit {
+     public static @Nullable ItemStack createExplorerMap(@NotNull World world, @NotNull Location location, @NotNull org.bukkit.generator.structure.StructureType structureType, @NotNull org.bukkit.map.MapCursor.Type mapIcon, int radius, boolean findUnexplored) {
+         return server.createExplorerMap(world, location, structureType, mapIcon, radius, findUnexplored);
+     }
++
++    /**
++     * Checks if the server is currently reloading.
++     *
++     * @return if the server is reloading
++     */
++    public static boolean isReloading() {
++        return server.isReloading();
++    }
```

Comment:
Create a separate `// Paper start - isReloading API` comment block instead of adding to an existing one. This is something I've been starting to do because it makes it more clear which patch diff comes from without having to look at the blame. Same in `Server` as well.